README.md

```diff
@@ -118,4 +118,8 @@ Create and run a new suite. Each suite creates and is associated with a `describ
   - **teardown**: run after each loop of benchmark. Note the caveat above.
   - **setupSuite**: run once before all benchmarks. This block, in effect, is the same as a `beforeAll` block (and it does call `beforeAll` underneath). Again you probably don't want to define or initialize variables here if you also include `setup` or `teardown`.
   - **teardownSuite**: run once after all benchmarks have concluded.
-- **timeout**: number of milliseconds before a benchmark timeout. Default to 60000.
+- **timeoutMsOrOptions**: 
+  - number of milliseconds before a benchmark timeouts. Default to 60000,
+  - or a `SuiteOptions`:
+    - `delay`, `initCount`, `maxTime`, `minSamples`, `minTime` are passed to Benchmark. See the [documentation](https://benchmarkjs.com/docs/#options) for more info.
+    - `timeoutSeconds` is the number of seconds before a benchmark timeouts. Default to 60.
```

Comment:
Looks good so far. Just one little kink though. `timeoutMsOrOptions` should be consistent regarding time unit for timeout. I see that you opted to use `timeoutSeconds`. Best change its unit to milliseconds to stay consistent with the old argument.

Reply:
See the test:

```
benchmarkSuite("string with SuiteOptions", {
  ["indexOf"]: () => {
    "Hello World!".indexOf("o") > -1;
  },

  ["match"]: () => {
    !!"Hello World!".match(/o/);
  },
}, { delay: 0.1, maxTime: 0.5, timeoutSeconds: 10 });
```

Notice how benchmark options are expressed in seconds.
It would be weird to mix seconds and milliseconds in the same object.
I named the key `timeoutSeconds` to try to alleviate confusion.

Reply:
You are right!